src/plugins/intel_cpu/src/nodes/interpolate.h

```diff
@@ -103,6 +103,13 @@ class Interpolate : public Node {
 private:
     bool is_version11 = true;
     InterpolateAttrs interpAttrs;
+    // Some FEs or preprocessing step resize spatial dimenion for tensor with NHWC layout memory,
+    // but imported as planar layout[abcd] with axis[1,2] for convinence. In this case, for pillow modes and no pad for now,
+    // nhwc layout pass and kernel can be used for this planar layout and axis settings(NCHWAsNHWC is true) to get higher perf with
+    // 1. logical shape alignment [abcd-nhwc] to [adbc-nchw].
+    // 2. axis alignment [1,2] to [2,3].
+    // 3. config planar layout support and treated it as channel_first layout.
```

Comment:
```suggestion
    // Some FEs or preprocessing step resize spatial dimmenion for tensor with NHWC layout memory,
    // but imported as planar layout[abcd] with axis[1,2] for convenience. In this case, for pillow modes without pad for now,
    // nhwc layout path and the kernel can be used for this planar layout and axis settings(NCHWAsNHWC is true) to get higher perf with
    // 1. logical shape alignment [abcd-nhwc] to [adbc-nchw].
    // 2. axis alignment [1,2] to [2,3].
    // 3. config planar layout support and treated it as channel_first layout.
```

Reply:
done in https://github.com/openvinotoolkit/openvino/pull/17246, thanks.